src/styles/blocks/content/card-container.scss

```
@@ -0,0 +1,17 @@
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: (4 * $card-width) + (3 * $card-gapX);
+  padding: 5px 0 70px;
```

it's better to set explicit top padding so that you don't need to combine it from multiple elments' styles
```suggestion
  padding: 50px 0 70px;
```
you'll then need to disable margin-top for the first 4 cards and you'll be good

---

We talked about all these line-heights Swazi mentioned in the review in DMs, my bad for not typing it out here, sorry.
Sadly they didn't help much.  Implemented the changes you recommended.